type_templates/light.js

```diff
@@ -85,6 +101,11 @@ export default e => {
       light.offsetMatrix = new THREE.Matrix4().makeTranslation(p.x, p.y, p.z);
       light.lastAppMatrixWorld = new THREE.Matrix4();
 
+      if(lightType === 'directional' || lightType === 'point' || lightType === 'spot') {
+        const s = (Array.isArray(shadow) && shadow.length === 5 && shadow.every(n => typeof n === 'number')) ?

```

Comment:
Minor: `s` is no longer used and `undefined` so this can be cleaned up to be an `if`.